ansible/roles/common/tasks/main.yml

```diff
@@ -23,6 +23,22 @@
     docker_config_dir: "/etc/default"
   when: ansible_distribution == "Debian" or ansible_distribution == "Ubuntu"
 
+- name: Create config file directory
+  file: path={{ docker_config_dir }} state=directory
+  when: ansible_lsb.id == "CoreOS"
```

Comment:
I added ansible_lsb.id == "CoreOS" b/c the dir creation task did not exist prior to my patch. I assumed that it already existed for other distro's and wanted to play it safe. I will remove the when: statement.